tools/run_tests/run_interop_tests.py

```diff
@@ -807,23 +807,17 @@ def compute_engine_creds_required(language, test_case):
     return False
 
 
-def auth_options(language,
-                 test_case,
-                 google_default_creds_use_key_file,
-                 service_account_key_file=None):
+def auth_options(language, test_case, google_default_creds_use_key_file,
```

Comment:
small nit: the vertical parameters are slightly neater

Reply:
I think this was done by yapf